js/itbit.js

```diff
@@ -203,7 +203,8 @@ module.exports = class itbit extends Exchange {
             'cost': cost,
         };
         if (feeCost !== undefined && rebatesApplied !== undefined) {
-            if (feeCurrency === rebateCurrency) {
+            if (feeCurrency === rebateCurrency || rebateCurrency === null
+                || typeof (rebateCurrency) === 'undefined') {
```

Comment:
This check is therefore also not needed here (for the same reason a few lines above). 
Also, syntactically, we use `rebateCurrency === undefined` instead (because it's shorter and much better than a string comparison).

Reply:
Ok. got it.  I thought I had experimented on undefined vs 'undefined' but that is a syntactic issue and I trust your judgment. 

Re: fees vs fee, we both agree that there needs to be two entries. I feel it is easier for a user of ccxt to always know that there will be a key named 'fee', but check for the length of the array. Which is why I made that change. However, whether one checks for 'fee' or 'fees' is essentially equivalent to looking for 'fee' but checking the number of records. Like I said, my personal inclination is that it is one fee, but paid in 2 currencies. But so what, your suggestion is equally valid (I think).

Reply:
Ok, can you help clean it up for `fees` + your change #1? (i mean, to make travis tests pass)